app.js

```diff
@@ -38,46 +43,54 @@ if (app.get("env") === "production") {
 app.use(session(sessionParms));
 app.use(flash());
 
-app.use((req, res, next) => {
-  res.locals.errors = req.flash("error");
-  res.locals.info = req.flash("info");
-  next();
-});
-
-app.post("/secretWord", (req, res) => {
-  if (req.body.secretWord.toUpperCase()[0] == "P") {
-    req.flash("error", "That word won't work!");
-    req.flash("error", "You can't use words that start with p.");
-  } else {
-    req.session.secretWord = req.body.secretWord;
-    req.flash("info", "The secret word was changed.");
-  }
-
-  // res.redirect("/secretWord");
-
-  res.render("secretWord", {
-    secretWord: req.session.secretWord,
-    errors: req.flash("error"),
-    info: req.flash("info"),
-  });
-});
-
-app.get("/secretWord", (req, res) => {
-  if (!req.session.secretWord) {
-    req.session.secretWord = "syzygy";
-  }
-
-  res.render("secretWord", {
-    secretWord: req.session.secretWord,
-    errors: req.flash("error"),
-    info: req.flash("info"),
-  });
-});
-
+// Initialize Passport
+passportInit(); // Run the passport initialization to register strategies
+app.use(passport.initialize());
+app.use(passport.session()); // Ensure session support for passport
+
+
+// Use the storeLocals middleware globally for all routes
+app.use(storeLocals);
+
+// Use the routes for /secretWord
+app.use("/secretWord", auth, secretWordRoutes);
+
+
+// // Routes for secretWord
+// app.post("/secretWord", (req, res) => {
+//   if (req.body.secretWord.toUpperCase()[0] === "P") {
+//     req.flash("error", "That word won't work!");
+//     req.flash("error", "You can't use words that start with p.");
+//   } else {
+//     req.session.secretWord = req.body.secretWord;
+//     req.flash("info", "The secret word was changed.");
+//   }
+
+//   res.render("secretWord", {
+//     secretWord: req.session.secretWord,
+//     errors: req.flash("error"),
+//     info: req.flash("info"),
+//   });
+// });
+
+// app.get("/secretWord", (req, res) => {
+//   if (!req.session.secretWord) {
+//     req.session.secretWord = "syzygy";
+//   }
+
+//   res.render("secretWord", {
+//     secretWord: req.session.secretWord,
+//     errors: req.flash("error"),
+//     info: req.flash("info"),
+//   });
+// });
```

Comment:
It doesn't really hurt to leave your old commented code, but you might as well just delete it completely once you have it moved to where you need it! You do have version control (on GitHub, or in git) to reference as well.